### PR TITLE
SW-1152-update-ledstrips-versioneer

### PR DESCRIPTION
**Pull request by JFSolution**
update versioneer and reformat setup.py file

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+# See the docstring in versioneer.py for instructions. Note that you must
+# re-run 'versioneer.py setup' after changing this section, and commit the
+# resulting files.
+
```
Comment (khaledsherkawi) [inline setup.cfg:5]: This is missing here:

`[metadata]

description-file = README.md`

File: setup.py [hunk 2]
```diff
@@ -1,173 +1,186 @@
 # coding=utf-8
-#!/usr/bin/env python
+# !/usr/bin/env python
 
 from setuptools import setup, Command
 import sys
```
Comment (khaledsherkawi) [inline setup.py:7]: `versioneer` is not imported here

**Review by khaledsherkawi — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (JFSolution) [reply setup.py:7]: ohh good catch, thx

File: setup.cfg (hunk 1 above)
Comment (JFSolution) [reply setup.cfg:5]: will add

**Review by khaledsherkawi — APPROVED** [review]

**Pull request merged**